velocity/src/main/java/me/leoko/advancedban/velocity/listener/ChatListenerVelocity.java

```diff
@@ -0,0 +1,24 @@
+package me.leoko.advancedban.velocity.listener;
+
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.player.PlayerChatEvent;
+import me.leoko.advancedban.Universal;
+
+public class ChatListenerVelocity {
+
+  @Subscribe
+  public void onChat(PlayerChatEvent event) {
+    if (!event.getMessage().startsWith("/")) {
+      if (Universal.get().getMethods().callChat(event.getPlayer())) {
+        event.setResult(PlayerChatEvent.ChatResult.denied());
+      }
+    } else {
+      if (Universal.get().getMethods().callCMD(event.getPlayer(), event.getMessage())) {
```

Comment:
else + if -> else-if
Even though you followed the example of ChatListenerBungee, it's better to introduce cleaner code going forward.

Reply:
Copied this and CommandRecieverVelocity from the old PR to save some time, after reading it again I'll be re-writing the entire thing